docs/source/initialsetup/configuringmame.rst

```diff
@@ -0,0 +1,78 @@
+Configuring MAME
+================
+
+.. contents:: :local:
+
+Getting Started: A Quick Preface
+================================
+
+Once you have MAME installed, the next step is to configure it. There are
+several ways to do this, and each will be covered in turn.
+
+If you are on Windows, the MAME executable will be called ``mame64.exe`` or
+``mame.exe`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+If you are on Linux or MacOS, the MAME executable will be called ``mame64`` or
+``mame`` depending on if you have a 64-bit or 32-bit version of MAME.
+
+
+Initial Setup: Creating MAME.ini From Command Line on Windows
+=============================================================
+
+First, you will need to *cd* to the directory where you installed MAME into.
+If, for instance, you have MAME installed in ``c:\emu\mame`` you will need to
+type ``cd \emu\mame`` into the command prompt.
+
+Then you have MAME create the config file by typing ``mame64 -createconfig``
+or ``mame -createconfig`` depending on if you are using the 64-bit version of
+MAME or the 32-bit version.
+
+MAME will then create the ``mame.ini`` file in the MAME installation folder.
+This file contains the default configuration settings for a new MAME
+installation.
+
+
+Initial Setup: Creating MAME.ini From Command Line on Linux or MacOS
+====================================================================
+
+The steps for Linux and MacOS are similar to those of Windows. If you
+installed MAME using the package manager that came from a Linux distro, you will
+type ``mame64 -createconfig`` or ``mame -createconfig`` into your terminal
+of choice, depending on if you have a 64-bit version of MAME or a 32-bit
+one.
+
+If you have compiled from source or downloaded a binary package of MAME,
+you will ``cd`` into the directory you put the MAME files into.
+
+For instance, ``cd /home/myusername/mame``
+
+Then you will type ``./mame64 -createconfig`` or ``./mame -createconfig`` into
+your terminal of choice, depending on if you have a 64-bit version of MAME or a
+32-bit one.
+
+You can then need to edit the ``mame.ini`` file in your favorite text editor,
+e.g. *notepad* on Windows or *vi* on Linux/MacOS, or you can change settings
+from inside of MAME.
+
+
+Initial Setup: Graphical Setup
+==============================
+
+This is the easiest way to get started. Start MAME by opening the MAME icon
+in the location where you installed it. This will be ``mame64.exe`` or
+``mame.exe`` on Windows, ``mame64`` or ``mame`` on Linux and MacOS.
```

Comment:
Hmm. Wasn't aware of that. Any suggestions?

Reply:
It’s a bit inconsistent because some of the Linux distributions provide MAME in a form that has a double-clickable shortcut in the desktop environment.

We describe a way to make a script that will change to the directory it resides in and start MAME on this page: https://docs.mamedev.org/commandline/commandline-all.html

Maybe it’s just not a great idea to say double-clicking is “easiest” on macOS/Linux, because in many cases it isn’t.  Changing to the directory where a command-line executable lives before launching it comes from the early days of Windows because a lot of DOS software expected to be launched that way, so it was a safe choice.  Most *NIX software expects to live under /usr or /opt so the assumption was never there.

Reply:
Maybe we should say it's the easiest way on Windows. For Mac and Linux, I'm not sure what to suggest. Maybe @rb6502 can suggest some better phrasing for Mac users? Then again, Mac shouldn't be that much different from Linux, right?

Reply:
Mac and Linux are pretty similar build-your-own or the various “download an archive” packages.  The distro-supplied packages are somewhat different because they modify MAME to integrate a bit more with their packaging and desktop environments.

Reply:
Okay, think I'm going to need to revisit this point with you after I get a look at everything else.